packages/teams-js/src/private/copilot.ts

```diff
@@ -0,0 +1,22 @@
+import { ensureInitialized } from '../internal/internalAPIs';
+import { runtime } from '../public/runtime';
+
+/**
+ * @beta
+ * User information required by specific apps
+ */
+export namespace copilot {
+  export namespace license {
```

Comment:
We mark all the tags on all levels: namespace, subnamespace, on top of every funciton etc